example/lib/main.dart

```diff
@@ -48,38 +87,213 @@ class _MyAppState extends State<MyApp> {
     return MaterialApp(
       home: Scaffold(
         appBar: AppBar(
+          backgroundColor: Color.fromRGBO(
+            141,
+            8,
+            1,
+            2,
+          ),
           title: const Text(
             StringConstants.titleAppBar,
           ),
         ),
-        body: Column(
-          children: [
-            Center(
-              child: ElevatedButton(
-                onPressed: () {
-                  setState(() {
-                    checkPermission();
-                  });
-                },
-                child: Text(
-                  StringConstants.checkPermissionButton,
-                ),
-              ),
+        body: Container(
+          decoration: BoxDecoration(
+            color: Color.fromRGBO(
+              178,
+              176,
+              155,
+              2,
             ),
-            ElevatedButton(
-              onPressed: () {
-                requestPermission();
-              },
-              child: Text(
-                StringConstants.requestPermissionButton,
+          ),
+          child: Center(
+            child: Container(
+              margin: new EdgeInsets.only(
```

Comment:
you can remove the "new" keyword, it's not necessary

Reply:
Fixed